### DISCUSSION TITLE
Vehicle comunication using Modem, where to get hardware?

### DISCUSSION THREAD

**Discussion by Xnygra** [dingo35/SmartEVSE-3.5#22] [General]
I thinking about build a modem into my SmartEvse to fetch the SoC from my Kia EV6 and Zoe 
The seller of the modem seems only to sell to other businesses, anyone have an alternative source, or have one for sale? 

https://github.com/ArendJanKramer/SmartEVSE-3/blob/modem_docs/manual/Modem.md
https://www.codico.com/en/yellow-beet-e-1-1

**Comment by dingo35** [comment]
No, dont have one.
